src/fiatExchanges/UniquePaymentSection.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+
+import { useTranslation } from 'react-i18next'
+import { Image, StyleSheet, Text, View } from 'react-native'
+import Touchable from 'src/components/Touchable'
+import { CICOFlow, FetchProvidersOutput, PaymentMethod } from 'src/fiatExchanges/utils'
+import colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+
+export interface UniquePaymentSectionProps {
+  paymentMethod: PaymentMethod
+  uniqueProvider: FetchProvidersOutput | null
+  setNoPaymentMethods: React.Dispatch<React.SetStateAction<boolean>>
+  flow: CICOFlow
+}
+
+export function UniquePaymentSection({
+  paymentMethod,
+  uniqueProvider,
+  setNoPaymentMethods,
+  flow,
+}: UniquePaymentSectionProps) {
+  const { t } = useTranslation()
+
+  if (!uniqueProvider || uniqueProvider.restricted) {
+    setNoPaymentMethods(true)
+    return null
+  }
+
+  return (
+    <View style={styles.container}>
+      <Touchable>
+        <View style={{ ...styles.innerContainer, paddingVertical: 27 }}>
+          <View style={styles.left}>
+            <Text style={styles.category}>
+              {paymentMethod === PaymentMethod.Coinbase
+                ? t('selectProviderScreen.coinbase')
+                : t('selectProviderScreen.somePaymentsUnavailable')}
```

Comment:
This is a very awkward fallback for the non-coinbase, unrecognized method case. "Some payment methods may be unavailable in your region" (or whatever this says) is supposed to appear when noPaymentMethods flips to true, not when we accidentally used this component for a provider we didn't decide on a payment method title for, and it's supposed to appear somewhere else on the page, not as a payment method category title. This weirdness seems like fallout from the decision to make the component agnostic to the actual method-- if it's explicitly coinbase-specific I think the problem would go away